src/mame/drivers/rainbow.cpp

```diff
@@ -2727,14 +2726,10 @@ READ16_MEMBER(rainbow_state::vram_r)
 // NOTE: Rainbow has separate registers for fore and background.
 WRITE16_MEMBER(rainbow_state::vram_w)
 {
-	if(!(m_GDC_MODE_REGISTER & GDC_MODE_VECTOR))
-	{
-		// SCROLL_MAP IN BITMAP MODE ONLY...?
-		if(m_GDC_MODE_REGISTER & GDC_MODE_HIGHRES)
+	if(m_GDC_MODE_REGISTER & GDC_MODE_HIGHRES)
 			offset = ( m_GDC_SCROLL_BUFFER[ (offset & 0x3FC0) >> 6 ] << 6) |  (offset & 0x3F);
-		else
+	else
```

Comment:
After the change, the controlled statements have double indent from the containing block.